tests/Helpers/UtilTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * This file is part of the php-resque package.
+ *
+ * (c) Michael Haynes <mike@mjphaynes.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\Resque\Heplers\Utils;
```

Comment:
I think the FQCN should be `Resque\Tests\Helpers\UtilTest` and it should be in the folder `tests/Helpers` to match the source folder structure. 

With this, autoload-rules can be set up to autoload the `Resque` namespace from `src` and autoload-dev the `Resque\Tests` namespace from `tests`. Using this approach, additional non-test classes (e.g. stub classes) can be autoloaded in development/testing, without the corresponding class loading overhead in production.